timeless/auth/views.py

```diff
@@ -75,7 +76,7 @@ def activate():
         return render_template("auth/activate.html")
 
     g.user.account_status = True
```

Comment:
@emilianodellacasa And then we don't need to store this flag in user model into DB.

Reply:
@vladarefiev I don't get it: whether a user is active should be saved to the DB and checked everytime he/she access to the webapp, otherwise how do we check if the admin deactiveated for any reason the user?

Reply:
@emilianodellacasa I've got idea. Ok.